chap-cheri-riscv.tex

```diff
@@ -260,7 +258,6 @@ \subsection{CHERI as a non-standard RISC-V extension}
 hardwired to 1 on boot to indicate the presence of a non-standard
 extension.  Information tying this set X bit to the Xcheri extension
 would be communicated to system software in a platform-specific manner.
-}
 \pdrnote{Agreed: this would be great, although pedantically, our adding of
```

Comment:
In particular, Peter's note here probably needs resolving one way or another?

Reply:
Humm, it seems like while most new instructions are in a new prefix, LC/SC and the atomics are both "brownfield" as you say.  I will try to describe this more fully.

Reply:
Great! Happy to delete my comment when I read through, or feel free to delete it now.